nixos/modules/services/monitoring/rustdesk-server.nix

```diff
@@ -18,30 +18,54 @@ in {
       '';
     };
 
-    relayIP = mkOption {
```

Comment:
We should add mkRenamedOption for the options which got removed and can be mapped to new ones and mkRemovedOption for those who cannot be mapped.